ncscolor.js

```diff
@@ -9,7 +9,7 @@
 
 })(this, function(){
 
-	var ncsRe = /^(?:NCS|NCS\sS)\s(\d{2})(\d{2})-(N|[A-Z])(\d{2})?([A-Z])?$/;
+	var ncsRe = /^(?:NCS|NCS\sS)\s(\d{2})(\d{2})-(N|R|G|B|Y)(\d{2})?([R|G|B|Y])?$/;

```

Comment:
I guess the second `R|G|B|Y` doesn't need to be in a `[]`-range anymore now, just like in the preceding part?